test/FastRouteRouterTest.php

```diff
@@ -823,4 +823,28 @@ public function testGetAllAllowedMethods()
             $result->getAllowedMethods()
         );
     }
+
+    public function testCustomDispatcherCallback()
+    {
+        $route1 = new Route('/foo', $this->getMiddleware());
+        $dispatcher = $this->prophesize(Dispatcher::class);
+        $dispatcher->dispatch(RequestMethod::METHOD_GET, '/foo')
+            ->shouldBeCalled()
+            ->willReturn([
+                Dispatcher::FOUND,
+                '/foo',
+                []
+            ]);
+
+        $router = new FastRouteRouter(null, function ($data) use ($dispatcher) {
```

Comment:
And one more, `$data` is not used, we can have here just:
```php
$router = new FastRouteRouter(null, [$dispatcher, 'reveal']);
```

Reply:
Nice one :smile: done.